Tests/Builders/InteractionTrainingBuilderTests.cs

```diff
@@ -25,9 +25,11 @@ protected override void InternalSetLocked(bool lockState)
             {
                 throw new NotImplementedException();
             }
-
+#pragma warning disable CS0067 // Disable "event is never used" warning.
```

Comment:
Should we simply remove these events?

Reply:
We can't, I think those are part of the interface.

Reply:
Not really, these are private classes.

Reply:
Those events are part of `ISnapZoneProperty`

![image](https://user-images.githubusercontent.com/6911992/84242359-0aa31b80-ab01-11ea-9361-6a6e0eb7e745.png)

Which these classes inherit from:

![image](https://user-images.githubusercontent.com/6911992/84242266-e3e4e500-ab00-11ea-82ac-f81faa8c1d3f.png)
![image](https://user-images.githubusercontent.com/6911992/84242303-f4955b00-ab00-11ea-9800-b83d29006ec0.png)